contributors/devel/sig-architecture/conformance-tests.md

```diff
@@ -70,6 +70,50 @@ reasonable production worthy environments:
   bypass policies that are not yet GA, but which may reasonably be enabled on a
   conformant cluster (e.g., pod security policy, non-GA scheduler annotations)
 
+### Windows & Linux Considerations
+
+As of v1.14, Windows is a stable feature, but not yet included in conformance
```

Comment:
I would change "not yet included in ... In preparation for this," to something like:

it is an optional feature and therefore not part of the conformance test suite. Nonetheless, it is important to verify that the behavior of Windows nodes conforms as much as possible. To that end,